### ISSUE TITLE
Safari version

### ISSUE THREAD

**Issue by Grawl** [ismamz/refined-wikipedia#2]
I am using Safari as my primary browser and I would be happy to have this extension. Or you can pack this as “user script” or “user style” to use with [Stylish](https://github.com/350d/stylish) and/or [Tampermonkey](https://tampermonkey.net/?browser=safari).

**Comment by ismamz** [comment]
Let me see what can I do. I'll try to find a solution for Safari and Firefox too!

**Comment by denysdovhan** [comment]
Looking forward for Safari version too.

**Comment by arguiot** [comment]
@Grawl Yes, you'll need an Apple Developer Membership, which allows you to publish an app, extensions, etc... on the App Store on the web for Safari Extension. So, yes, it's 100$, but it worth it. Also, you don't even need an Apple Developer membership to develop the extension, you'll just need it if you want to publish it, so, with a little manipulation, I think you can publish it on GitHub.

**Comment by ismamz** [comment]
Maybe we need to find someone who already has an Apple Developer Membership 😕

**Comment by Grawl** [comment]
Maybe better to make it a userscript/userstyle?

**Comment by gingerbeardman** [comment]
I'll happily sign it for Safari.

**Comment by ismamz** [comment]
@gingerbeardman Hi!, Can you adapt this extension for Safari and publish? It would be great!
